.stylelintrc.json

```
@@ -4,6 +4,20 @@
 		"at-rule-empty-line-before": null,
 		"at-rule-no-unknown": null,
 		"comment-empty-line-before": null,
+		"declaration-property-value-allowed-list": [
+			{
+				"flex-direction": [
+					"row",
+					"column",
+					"inherit",
+					"initial",
+					"unset"
```

We should be able to do it with regex. I think it might be easier if we just list all the possible reverse values and with a negative regex (untested):



```suggestion
					"/-reverse$/",
```

That will also allow using variables as @youknowriad raised above.

---

@tyxla sure but in order to do that we should use `declaration-property-value-disallowed-list` which is already used. It's not clear to me how to add multiple disallowed properties with custom messages under the same Stylelint rule. Any clue?

---

> add multiple disallowed properties with custom messages under the same Stylelint rule

This is where we need to first convert the config file from JSON to JS (#63090).

---

+1 to converting it. @mirka would you advise to wait for the conversion to happen or maybe proceed with this PR as is, and then iterate when https://github.com/WordPress/gutenberg/issues/63090 will be solved?

---

> @tyxla sure but in order to do that we should use `declaration-property-value-disallowed-list` which is already used. It's not clear to me how to add multiple disallowed properties with custom messages under the same Stylelint rule. Any clue?

Good point, but I think you should be able to do it with `declaration-property-value-allowed-list`, just need to invert the regex to match anything BUT `row-reverse` and `column-reverse`, for example:

```
"/^(?!(row|column)-reverse).*$/"
```

Haven't tested it, so let me know if that works for you.

---

Otherwise, converting to the JS format makes sense, although it doesn't look like a high-priority task.